x-pack/plugins/session_view/public/components/detail_panel_list_item/styles.ts

```diff
@@ -36,7 +36,7 @@ export const useStyles = ({ display }: StylesDeps) => {
       '&:hover': {
         background: transparentize(euiTheme.colors.primary, 0.1),
       },
-      height: '100%',
+      height: 'fit-content',
```

Comment:
If the height is `fit-content` here, is there still a gap around the copy container like in the design here?
<img width="545" alt="image" src="https://user-images.githubusercontent.com/16872649/163031086-920d0809-a767-4cf2-bb0f-dcf1edbae324.png">
I imagine we will need some paddings instead